game/src/engine/config/ConfigService.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <filesystem>
+
+#include <yaml-cpp/yaml.h>
+#include "engine/service/Service.h"
+#include "VehicleData.h"
+
+namespace fs = std::filesystem;
+class ConfigService final : public Service
+{
+    public:
+        ConfigService();
+        void OnInit() override;
+        void OnStart(ServiceProvider& service_provider) override;
+        void OnUpdate() override;
+        void OnCleanup() override;
+        VehicleData getVehicleData();
```

Comment:
styling nitpick, but could this be changed to `GetVehicleData` to match the other public methods?